packages/server/core/src/core/auth/internal/StitchAuthImpl.ts

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright 2018-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  AuthInfo,
+  CoreStitchAuth,
+  CoreStitchUser,
+  DeviceFields,
+  StitchAppClientInfo,
+  StitchAuthRoutes,
+  StitchClientError,
+  StitchClientErrorCode,
+  StitchCredential,
+  StitchRequestClient,
+  StitchUserFactory,
+  Storage
+} from "mongodb-stitch-core-sdk";
+
+import version from "../../internal/common/Version";
+import AuthProviderClientFactory from "../providers/internal/AuthProviderClientFactory";
+import NamedAuthProviderClientFactory from "../providers/internal/NamedAuthProviderClientFactory";
+import StitchAuth from "../StitchAuth";
+import StitchAuthListener from "../StitchAuthListener";
+import StitchUser from "../StitchUser";
+import StitchUserFactoryImpl from "./StitchUserFactoryImpl";
+
+/**
+ * Partial JSDOM window.location containing only the
+ * properties and method we need
+ */
+interface PartialLocaion {
```

Comment:
[nit] assuming this same typo exists in browser, you should probably drive-by fix it there as well as delete here.